src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colour colour;
+    private int number;
+
+    public Ball(Colour colour, int number) {
```

Comment:
The constructor for `Ball` should also have an explicit access modifier to define its visibility clearly.

Reply:
Fixed